src/test/java/com/github/webdriverextensions/newversion/FileExtractorImplTest.java

```diff
@@ -260,4 +260,20 @@ public void extractFile_should_extract_zip_file_with_extract_pattern() throws Ex
         assertThat(toDirectory.toFile().listFiles().length, is(1));
         assertThat(toDirectory.resolve("a-file-in-directory.txt").toFile().exists(), is(true));
     }
+
+    @Test
+    public void test_unZipOldMS() throws Exception {
+        Path aNewZip = Paths.get("src/test/resources/file-extractor-test-data/edge80.zip");
+        FileExtractorImpl fileExtractor = new FileExtractorImpl(null);
+        fileExtractor.extractFile(aNewZip,toDirectory);
+        assert(true);
```

Comment:
That assertion should be changed because it will always pass. There is no test that compares the actual behavior against the expected behavior. The test should fail if the changes in `FileExtractorImpl` are undone and it should pass if those changes are applied.

Reply:
The test fails if FileExtractorImpl throws an exception, as intended

Reply:
Then you might want to use `org.junit.Assert.assertThrows` and test for the expected Exception.

Reply:
Although you are right and a test fails by default if it throws an exception, it is not clear from reading the code that this is intended and part of the actual test. Making it more explicit helps to understand the intention of the test.

Reply:
Agree with "assertThrows".